File: root/common/src/main/java/org/sc/common/rest/AccessibilityUnresolvedResponse.java
package org.sc.common.rest;

import java.util.Collections;
import java.util.List;

public class AccessibilityUnresolvedResponse extends RESTResponse {
    List<AccessibilityNotificationUnresolved> accessibilityNotifications;

    public AccessibilityUnresolvedResponse(){}

    public AccessibilityUnresolvedResponse(final List<AccessibilityNotificationUnresolved> accessibilityNotifications) {
        super(Status.OK, Collections.emptySet());
        this.accessibilityNotifications = accessibilityNotifications;
    }

    public List<AccessibilityNotificationUnresolved> getAccessibilityNotifications() {
        return accessibilityNotifications;
    }
}

File: root/common/src/main/java/org/sc/common/rest/response/AccessibilityUnresolvedResponse.kt
package org.sc.common.rest.response

import org.sc.common.rest.AccessibilityUnresolvedDto
import org.sc.common.rest.Status

data class AccessibilityUnresolvedResponse (
    val status: Status,
    val messages: Set<String>,
    val accessibilityNotifications : List<AccessibilityUnresolvedDto>)
    : RESTResponse()